src/CompositeRheologies.jl

```diff
@@ -132,6 +138,10 @@ function show(io::IO, a::Parallel)
     return nothing
 end
 
+function isvolumetric(c::NTuple{N,Any}) where {N}
```

Comment:
might be slightly better to define this function as `function isvolumetric(c::NTuple{N,T}) where {N,T<:AbstractCreepLaw}` - or whatever the appropriate type is.

Reply:
in fact, we won't even need this function as we can use broadcasting:
```Julia
isvolumetric.(xxx)
```